force-app/main/default/classes/AccountHelper.cls

```diff
@@ -1,46 +1,77 @@
-
-The `AccountHelper` class is designed to provide helper methods that assist in performing specific operations on Account records. 
-This class is intended to modularize the logic used in the `AccountTrigger` to ensure separation of concerns, improve readability, and ease maintenance. 
-
+    
+The `AccountHelper` class is designed to provide helper methods that assist in performing specific operations on Account records.
+    This class is intended to modularize the logic used in the `AccountTrigger` to ensure separation of concerns, improve readability, and ease maintenance.
+    
 This class is currently stubbed out. As part of this exercise, you are expected to:
-1. Implement the method bodies to carry out the specified operations on Account records.
-2. Refactor the `AccountTrigger` to utilize the methods provided in this `AccountHelper` class.
-
+    1. Implement the method bodies to carry out the specified operations on Account records.
+    2. Refactor the `AccountTrigger` to utilize the methods provided in this `AccountHelper` class.
+    
 Methods:
-- `setTypeProspect(SObject objects)`: 
+    - `setTypeProspect(SObject objects)`:
     - Purpose: Update the 'Type' field of the given Account record to 'Prospect' if it's not already set.
     - Note: You'll likely need to adjust the method signature to accept the necessary parameters.
-
-- `addressCopy()`: 
-    - Purpose: Copy the shipping address of the Account to its billing address. 
+    
+- `addressCopy()`:
+    - Purpose: Copy the shipping address of the Account to its billing address.
     - Note: You'll likely need to adjust the method signature to accept the necessary parameters.
-
-- `setRating`: 
+    
+- `setRating`:
     - Purpose: Evaluate certain conditions on the Account and assign a rating based on that evaluation. You'll need to define the method's signature and implement its logic.
-
-- `defaultContact`: 
+    
+- `defaultContact`:
     - Purpose: Create a default contact related to the Account after the Account is inserted. You'll need to define the method's signature and implement its logic.
-
+    
 Remember:
-Always ensure that your helper methods are bulk-safe to handle multiple records efficiently. 
-*/
+    Always ensure that your helper methods are bulk-safe to handle multiple records efficiently.
+     */
+
+
 public with sharing class AccountHelper {
+    // Method to set the Account Type to 'Prospect' if it is not already set
+    public static void setTypeProspect(List<Account> accounts) {
+        for (Account acc : accounts) {
+            if (acc.Type == null) {
+                acc.Type = 'Prospect';
+            }
+        }
+    }
+    // Method to copy the shipping address to the billing address if shipping fields are not null
+    public static void addressCopy(List<Account> accounts) {
+        for (Account acc : accounts) {
+            if (acc.ShippingStreet != null && acc.ShippingCity != null && acc.ShippingState != null &&
```

Comment:
Consider doing a null check on each individual field before setting it rather than only setting if all have values